src/main/java/org/broadinstitute/hellbender/tools/walkers/rnaseq/OverhangFixingManager.java

```diff
@@ -457,6 +463,12 @@ public int compare(final List<SplitRead> readgroup1, final List<SplitRead> readg
         }
     }
 
+    public SplitRead getSplitRead(final GATKRead read){
```

Comment:
Why did you pull these out into class methods?  It seems more readable to just call the constructors like you were before.  Maybe I'm missing some use of them.  Can you at least make them static if you need them?

Reply:
Again I needed them for testing, and I cant actually make them static unfortunately, because they rely on the inner classes which cant be made static because they use the managers dictionary

Reply:
Oh, gross, I didn't see that.